bool running
uint16 red_score
uint16 green_score
uint16 left_time
sensor_msgs/Image color_map